### PR TITLE
Update code using new css class

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

**Comment by dolibarr95** [comment]
I wonder how many `$var=true;` we have to delete...

File: htdocs/admin/compta.php [hunk 1]
```diff
@@ -119,7 +119,7 @@
 print '<tr class="liste_titre">';
 print '<td>'.$langs->trans('OptionMode').'</td><td>'.$langs->trans('Description').'</td>';
 print "</tr>\n";
-print '<tr '.$bc[false].'><td width="200"><input type="radio" name="accounting_mode" value="RECETTES-DEPENSES"'.($accounting_mode != 'CREANCES-DETTES' ? ' checked' : '').'> '.$langs->trans('OptionModeTrue').'</td>';
+print '<tr class=\"oddeven\"><td width="200"><input type="radio" name="accounting_mode" value="RECETTES-DEPENSES"'.($accounting_mode != 'CREANCES-DETTES' ? ' checked' : '').'> '.$langs->trans('OptionModeTrue').'</td>';
```
Comment (eldy) [inline htdocs/admin/compta.php]: Wrong syntax here. No \ required when inside ''

File: htdocs/admin/agenda_other.php [hunk 2]
```diff
@@ -266,7 +265,7 @@
             			$module = new $classname($db, new ActionComm($db));
             			
             			
-            			print "<tr ".$bc[$var].">\n";
+            			print '<tr class="oddeven">\n';
```
Comment (eldy) [inline htdocs/admin/agenda_other.php]: No \n into simple quotes

**Comment by grandoc** [comment]
Le 04/05/2017 à 21:00, Laurent
      Destailleur a écrit :
    
    
      @eldy commented on this pull request.
      
      In htdocs/admin/agenda_other.php:
      > @@ -266,7 +265,7 @@
             			$module = new $classname($db, new ActionComm($db));
             			
             			
-            			print "<tr ".$bc[$var].">\n";
+            			print '<tr class="oddeven">\n';

      No \n into ''
      —
        You are receiving this because you authored the thread.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Dolibarr/dolibarr","title":"Dolibarr/dolibarr","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Dolibarr/dolibarr"}},"updates":{"snippets":[{"icon":"PERSON","message":"@eldy commented on #6738"}],"action":{"name":"View Pull Request","url":"https://github.com/Dolibarr/dolibarr/pull/6738#pullrequestreview-36358107"}}}
    
    
    -- 
      
        
          
            
              
              
            
            
              Philippe GRAND
                    ATOO.NET
                    SIREN 432 991 636 - APE 6202A
                    E-mail : philippe.grand@atoo-net.com
                    GSM : 06 80 22 33 18 
                    Tel : 02 38 63 90 20
                    Web : https://www.atoo-net.com

**Pull request merged**